src/main.py

```
@@ -194,7 +196,8 @@ def submit_feedback():
 
     except Exception as e:
```

Pylint (4.0.4)

[warning] Catching too general exception Exception

(W0718)